tests/serde.rs

```diff
@@ -114,7 +115,7 @@ fn test_ser_regex() {
 
     let regex = Regex {
         pattern: "12".into(),
-        options: "10".into(),
+        options: "01".into(),
```

Comment:
Since regex options are now alphabetized, the easiest way to fix these tests was to specify already-alphabetized options. These tests are just checking serde, so we can rely on the corpus tests to validate the alphabetization behavior.